repo/plugin.video.nzbdav/resources/lib/router.py

```
@@ -2162,13 +2184,44 @@ def _json_object(response):
+    so a hostile payload falls through to the empty-string fallback.
+    """
+    try:
+        from defusedxml import ElementTree as element_tree
+    except ImportError:  # pragma: no cover - Kodi installs may not bundle defusedxml
+        from xml.etree import ElementTree as element_tree
 
     try:
-        root = ET.fromstring(response)  # nosec B314 - trusted service response
-    except (TypeError, ET.ParseError):
+        if getattr(element_tree, "__name__", "").startswith("defusedxml."):
+            root = element_tree.fromstring(response)
```

Ruff (0.15.20)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)